src/main/webapp/student/projectSubmissions/viewProjectsWithOnlineSubmission.jsp

```diff
@@ -47,12 +47,14 @@
 			<ul>
 				<logic:iterate id="projectWithOnlineSubmission" name="projectsWithOnlineSubmission" type="org.fenixedu.academic.domain.Project">
 					<bean:define id="projectId" name="projectWithOnlineSubmission" property="externalId" />
-					<li>
-						<bean:write name="projectWithOnlineSubmission" property="name"/> , 
-						<html:link action="<%="/projectSubmission.do?method=viewProjectSubmissions&amp;attendsId=" + attendsId + "&amp;projectId=" + projectId%>">
-							<bean:message key="link.projectSubmissions.viewProjectsWithOnlineSubmission.viewProjectSubmissions"/>
-						</html:link>
-					</li>
+					<logic:notEqual value="false" name="projectWithOnlineSubmission" property="visible">
```

Comment:
Double negatives are not that intuitive for who is reading :) You can read a little bit more about it here: https://refactoring.com/catalog/removeDoubleNegative.html
Do you think it is better this way?
```suggestion
					<logic:equal value="true" name="projectWithOnlineSubmission" property="visible">
```

Reply:
I had that initially. However, it made so that the projects with the default value (`NULL`) were hidden, which would affect all existing projects.
Would there be a better way to migrate existing projects to have the `true` value by default?

Another option that I have thought, would be to instead rename this new property to `archive`, which might be more intuitive and solve the migration problem.

Reply:
I think you've fixed that in the `getVisible` method of the bean right? I believe it would be better to handle the null value by overriding the `getVisible` method in the domain class `Project`. The returned Boolean will never be null.
The migration is another possibility.
Nevertheless, this suggestion still applies, right? Or is there any quirk I am not recalling? 😅

Reply:
Yes, I'll definitely look into that!  
Thanks a lot for the feedback :+1: